runtime/doc/nvim-extra-keymaps.txt

```diff
@@ -0,0 +1,36 @@
+*nvim-extra-keymaps.txt*    For nvim.                         {Nvim}
+						    Last change:      2015 May
+
+EXTRA KEYMAPS FOR NVIM                                      *nvim-extra-keymaps*
+
+The `plugin/extra-keymaps.vim` script provides a few extra keymaps that might
+be useful and tries to fix some issues the community has identified with vim's
+default keymaps.
+
+The user can disable the plugin by having >
+
+    let g:nvim#defaults#maps#use = 0
+
+in their |vimrc| (additionally, all features can be disabled separately,
+using the variables described below).
+
+============================================================================
```

Comment:
In the other instances of `nvim_*.txt` I looked at, this  separator expects something like the following scheme:
```
==============================================================================
1. Configuration					    *nvim-configuration*
```

Given how small this file is, I think it can just be removed (like in nvim_clipboard.txt, where I think the lack of an index works well given its size).